web/src/main/webapp/xslt/ui-metadata/form-configurator.xsl

```diff
@@ -341,6 +341,7 @@
           <xsl:if test="($nonExistingChildParent/* and not(@ifNotExist)) or
             ($nonExistingChildParent/* and count($nodes/*) = 0 and @ifNotExist)">
             <xsl:variable name="childName" select="@or"/>
+            <xsl:variable name="xpath" select="@xpath"/>
```

Comment:
`@xpath` attribute in a field configuration is not exactly the same as the `context` attribute in `labels.xml`. It can use more advanced filters and complex XPath expression. `context` in `labels.xml` is "Parent element name or full XPath" ie. XPath with no expression, no *, ... 

Using `@xpath` only works in some cases.
 
So instead I would use something like 
```
<xsl:variable name="xpath" select="gn-fn-metadata:getXPath(gn-fn-metadata:getOriginalNode($metadata, $nodes/*[1]))"/>
```
Tested locally quickly and looks to be ok. I let you double check and if it works for you, you can update your PR. Thanks.